docs/open-api/openapi.yaml

```diff
@@ -77,6 +77,9 @@ paths:
   /metalakes/{metalake}/tags/{tag}/objects:
     $ref: "./tags.yaml#/paths/~1metalakes~1%7Bmetalake%7D~1tags~1%7Btag%7D~1objects"
 
+  /metalakes/{metalake}/tags/objects:
```

Comment:
I realize that this request path may conflict with getting a tag? We may need to define a new path.

The following are two solutions that come to my mind:
1. Treat `objects` as a reserved tag name, which may break compatibility. (If the user already has a tag named `objects`, it cannot be retrieved anymore)
2. using `GET /metalakes/{metalake}/objects?tags=tag1,tag2`

I prefer the solution 2, WDYT?

Reply:
cc @jerryshao  @jerqi